src/rgw/rgw_op.h

```diff
@@ -1339,6 +1339,19 @@ class RGWDeleteMultiObj : public RGWOp {
   virtual uint32_t op_mask() { return RGW_OP_TYPE_DELETE; }
 };
 
+class RGWInfo: public RGWOp {
+public:
+  RGWInfo() {}
+
+  void execute() {};
+  int verify_permission() { return 0; }
+
+  virtual void send_response() = 0;
+  virtual const string name() { return "get info"; }
+  virtual RGWOpType get_type() { return RGW_OP_GET_INFO; }
+  virtual uint32_t op_mask() { return RGW_OP_TYPE_READ; }
```

Comment:
`send_response()` is already a pure virtual in `RGWOp`, so you shouldn't need it here. for the others, can you use the `override` keyword instead of `virtual`?